cypress/integration/Select.spec.ts

```diff
@@ -353,6 +353,17 @@ describe('Select', () => {
         cy.findByText('Selected Value: Phone');
       });
     });
+
+    context('when spacebar is types and no value exists', () => {
+      beforeEach(() => {
+        cy.findByRole('combobox').focus();
+        cy.focused().realType(' ');
+      });
+      it('should select phone and the selected value should be visible', () => {
+        // should open the menu
+        cy.findByRole('listbox');
```

Comment:
I don't like implicit assertions. Add the assertion to be explicit about intent.

```suggestion
        cy.findByRole('listbox').should('exist');
```